index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio: Benjamin Beeksma</title>
+    <meta name="viewport" contente="width=device-width, intial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Goudy+Bookletter+1911|Limelight|News+Cycle|Oswald" rel="stylesheet">
+    <link rel="stylesheet" href="vendor/styles/reset.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://use.fontawesome.com/aef57db9f9.js"></script>
+  </head>
+  <body>
+    <header class="siteHeader clearfix">
+      <nav class="nav">
+        <ul class ="siteNav">
+          <li class ="ellipsis-v"><span class="fa fa-ellipsis-v" aria-hidden="true"></span></li>
```

Comment:
UX-wise, this may be a little too small to be useful. Maybe add some padding to increase the click target?